test/test_helper.rb

```diff
@@ -0,0 +1,13 @@
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
We also forgot this,  we were adviced to delete the test directory as we use rspec